packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -471,6 +474,11 @@ class ListView extends BoxScrollView {
   ///
   /// [itemBuilder] will be called only with indices greater than or equal to
   /// zero and less than [itemCount].
+  ///
+  /// The builder should actually create the widget instances when called. Avoid
```

Comment:
Changed both to `The 'itemBuilder'` (with backticks) since it's not actually something hyperlinkable.